pub_mobility_boundary_topic: mobility_boundary
kMobilityBoundaryFilename: mobility_boundary.ply
